cmd/root.go

```diff
@@ -304,11 +304,11 @@ func getConfig(nodeInfo node.Info) (interfaces.Config, error) {
 	if err != nil {
 		return nil, err
 	}
-	logWithCommand.Infof("configuring service for database type: %s", dbType)
+	logWithCommand.Infof("Configuring service for database type: %s", dbType)
```

Comment:
Good typography, but in peeve-land ideally better to do this in a separate PR (for future).

Reply:
Will do in the future, this PR felt small enough to sneak it in without making the review much more painful

Reply:
np. The peeve isn't with review overhead, more with preserving clean deployed code provenance. If something breaks in production and you track it back to a specific PR, you don't want extra changes in the PR that were unrelated to its purpose because now you have to figure out which of several changes caused the problem.